resources/build.gradle.kts

```diff
@@ -60,3 +64,10 @@ val copyIosArm64TestResources = tasks.register<Copy>("copyIosArm64TestResources"
 }
 
 tasks.findByName("iosSimulatorArm64Test")!!.dependsOn(copyIosArm64TestResources)
+
+afterEvaluate {
+    tasks.withType<JavaCompile>().configureEach {
+        sourceCompatibility = JavaVersion.VERSION_17.toString()
+        targetCompatibility = JavaVersion.VERSION_17.toString()
+    }
+}
```

Comment:
i think this is not necessary. also addition of this block to each build gradle not better solution - use convention plugin please